pkg/hns/netconf_windows.go

```diff
@@ -28,13 +28,35 @@ type NetConf struct {
 	types.NetConf
 	HcnPolicyArgs []hcn.EndpointPolicy `json:"HcnPolicyArgs,omitempty"`
 	Policies      []policy             `json:"policies,omitempty"`
+	RuntimeConfig RuntimeConfig        `json:"runtimeConfig"`
```

Comment:
If you want the runtime to pass the runtime config, `capabilityArgs` needs to be passed in the CNI configuration. You should update your plugin's README so users add it.

Reply:
Added this into the readme